src/Components/PaperEditor/TranscriptsContainer/TranscriptTabPane/TranscriptTabContent/index.js

```diff
@@ -458,6 +453,15 @@ const TranscriptTabContent = (props) => {
     props.trackEvent({ category: 'paperEditor transcriptsTab', action: `label select ${ labelId }` });
   };
 
+  const onLabelCreate = (newLabel) => {
+    createLabel(newLabel);
+    const tempLabels = labels;
+    newLabel.active = true;
+    tempLabels.push(newLabel);
```

Comment:
We've lost the `setLabels` state update here - there are a few little bugs and I'm wondering if not setting the labels immediately in state might be causing them? Mainly around the text temporarily disappearing after highlighting.... so might be related and might not be